File: Dart/OOP/classes&Objects/program2.dart
class Demo {
  int x = 10;
  static int y = 20;

  void printData() {
    print(x);
    print(y);
  }
}

void main() {
  Demo obj1 = new Demo();
  obj1.printData();

  Demo obj2 = Demo();
  obj2.printData();
  print("==============");

  obj1.y = 50;
  obj1.printData();
  obj2.printData();
}
/*Error: The setter 'y' isn't defined for the class 'Demo'.
 - 'Demo' is from 'Dart/OOP/classes&Objects/p2.dart'.    
Try correcting the name to the name of an existing setter, or defining a setter or field named 'y'.
  obj1.y = 50;*/

File: JAVA/OOP/OOP revision/src/com/OOP/Static/program2.java
package com.OOP.Static;

public class program2 {

    static  int x=50;
    String str="Tejas";


    static class innerClass{
      //  static int x=50;
        String str;
        public innerClass(String str) {
            this.str = str;
        }

        @Override
        public String toString() {
            return "object of innerclass";
        }
    }


    public static void main(String[] args) {

        innerClass obj1=new innerClass("Tejas");
        innerClass obj2=new innerClass("Tejas");
        System.out.println(123456);
        obj2=null;

        System.out.println(x);
        System.out.println(obj1);
        System.out.println(obj2);


    }
}
